config/totem.conf.example

```diff
@@ -62,24 +62,28 @@ totem {
       uri = ["http://127.0.0.1:7710/objdump/"]
       resultRoutingKey = "objdump.result.static.totem"
     }
+    passivetotal {
+      uri = ["http://127.0.0.1:7720/passivetotal/"]
+      resultRoutingKey = "passivetotal.result.static.totem"
+    }
     peid {
-      uri = ["http://127.0.0.1:7720/peid/"]
+      uri = ["http://127.0.0.1:7730/peid/"]
```

Comment:
revert the ports please as it should be 97xx and wouldn't require these to change.